### PR TITLE
Replaced KResponsiveWindow mixin by useKResponsiveWindow composable - …

### LINKED ISSUE #11327: Replace KResponsiveWindow mixin by useKResponsiveWindow composable - Media Player plugin
This is one of the issues whose common goal is to gradually replace `KResponsiveWindow` mixin by `useKResponsiveWindow` composable.

First, see https://github.com/learningequality/kolibri/issues/11321 for detailed instructions and acceptance criteria common to all issues.

Then, replace `KResponsiveWindow` mixin by `useKResponsiveWindow` composable in all .vue files of **the Media Player plugin (/kolibri/plugins/media_player)**

### PR DESCRIPTION
**Pull request by FireSuperior482**
…Media Player plugin

## Testing checklist

- [X] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [ ] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added.
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: kolibri/plugins/media_player/assets/src/views/MediaPlayerIndex.vue [hunk 1]
```diff
@@ -113,7 +113,11 @@
   export default {
     name: 'MediaPlayerIndex',
     components: { MediaPlayerFullscreen, MediaPlayerTranscript },
-    mixins: [commonCoreStrings, responsiveWindowMixin, responsiveElementMixin],
+    mixins: [commonCoreStrings, responsiveElementMixin],
+    setup() {
+      const { windowIsSmall } = useKResponsiveWindow();
```
Comment (rtibbles) [inline kolibri/plugins/media_player/assets/src/views/MediaPlayerIndex.vue]: The `windowIsPortrait` property is also used, so must be destructured here and returned from the setup method.

See here: https://github.com/learningequality/kolibri/blob/release-v0.16.x/kolibri/plugins/media_player/assets/src/views/MediaPlayerIndex.vue#L207

**Review by rtibbles — CHANGES_REQUESTED** [review]
One property missed!

File: kolibri/plugins/media_player/assets/src/views/MediaPlayerIndex.vue (hunk 1 above)
Comment (FireSuperior482) [reply kolibri/plugins/media_player/assets/src/views/MediaPlayerIndex.vue]: I have made the necessary changes; please check it once.

**Review by rtibbles — APPROVED** [review]
This doesn't cause any errors in manual testing, and is the change I expect.

**Pull request merged**